common/lib/capa/capa/tests/test_html_render.py

```diff
@@ -27,8 +27,6 @@ def test_blank_problem(self):
 
         # Render the HTML
         etree.XML(problem.get_html())
-        # expect that we made it here without blowing up
-        self.assertTrue(True)
```

Comment:
It really doesn't test much though. It would be better to like, instantiate the XML as further problems do (`rendered_html = etree.XML(problem.get_html())`) and then actually assert something about the rendered_html itself.

I'm fine if you don't want to do that here, but if you don't want to do that, please add a TODO eg `TODO: This test should inspect the rendered html and assert one or more things about it`